client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -415,6 +415,28 @@ function mediaButton( editor ) {
 		}
 	} );
 
+	editor.addButton( 'wp_img_edit', {
+		tooltip: 'Edit',
```

Comment:
Missing a translate here:
```
i18n.translate( 'Edit' )
```

^^ Directly calling translate from i18n is uncommon, see also: https://github.com/Automattic/i18n-calypso/#i18n-calypso for more info on our higher-order component and legacy mixin.